test/utils/executor.hpp

```diff
@@ -151,18 +127,22 @@ class CommonTestFixture : public ::testing::Test {
 
     CommonTestFixture()
         :
-#if defined(GKO_TEST_NONDEFAULT_STREAM) && \
-    (defined(GKO_COMPILING_CUDA) || defined(GKO_COMPILING_HIP))
-          stream{0},
+#if defined(GKO_TEST_NONDEFAULT_STREAM) && defined(GKO_COMPILING_CUDA)
+          stream(ResourceEnvironment::cuda_device_id),
+#endif
```

Comment:
when test default stream, it will use the default constructor of stream which is the default stream, right?

Reply:
exactly